azurerm/internal/services/kusto/tests/kusto_iothub_data_connection_resource_test.go

```diff
@@ -0,0 +1,101 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccAzureRMKustoIotHubDataConnection_basic(t *testing.T) {
```

Comment:
Could we add a complete test which sets _all_ optional properties and then an update test that goes basic -> complete -> basic?